src/libs/actions/PersonalDetails.js

```diff
@@ -420,6 +421,44 @@ function clearAvatarErrors() {
     });
 }
 
+/**
+ *
+ * @param {Object} detail
+ * @returns {Object}
+ */
+function buildNewPersonalDetail(detail) {
+    const login = Str.isValidPhone(detail.login) ? OptionsListUtils.addSMSDomainIfPhoneNumber(detail.login).toLowerCase() : detail.login.toLowerCase();
```

Comment:
Any specific reason why we need the `Str.isValidPhone` check? Because `addSMSDomainIfPhoneNumber` already uses `Str.isValidPhone` and returns the arg as fallback?